crypto/3des/3des-file-encrypt.c

```diff
@@ -255,13 +255,16 @@ int NoEcho(char* key, int size)
     }
 
     printf("Unique Password: ");
-    fgets(key, size, stdin);
+    if (fgets(key, size, stdin) == NULL) {
+        printf("Error\n");
```

Comment:
Please make this a more descriptive error than "Error".  It should indicate to the user that fgets() has failed.